### PR TITLE
[MU4] Crash after adding a note, saving, then changing its pitch

### PR DESCRIPTION
**Pull request by Eism**
Resolves: #13159

### CODE REVIEW

**Comment by DmitryArefiev** [comment]
Tested #13159 on Win10, LinuxMint 20.2 - **FIXED**

File: src/engraving/libmscore/engravingitem.cpp [hunk 1]
```diff
@@ -176,6 +177,18 @@ AccessibleItemPtr EngravingItem::createAccessible()
...
 
+void EngravingItem::notifyAboutNameChanged()
+{
+    if (!selected()) {
+        return;
+    }
+
+    if (m_accessible) {
+        doInitAccessible();
+        m_accessible->accessibleRoot()->notifyAboutFocuedElemntNameChanged();
```
Comment (MarcSabatella) [inline src/engraving/libmscore/engravingitem.cpp:188]: "notifyAboutFocuedElemntNameChanged"?  Is that the name of one of our functions?  Two typos for the price of one :-)

**Comment by MarcSabatella** [comment]
Justed tested the current artifact (the one that passed the tests!) and am happy to see that it too seems to work well.

File: src/engraving/libmscore/engravingitem.cpp (hunk 1 above)
Comment (Eism) [reply src/engraving/libmscore/engravingitem.cpp]: Ah, I was rushing

**Comment by cbjeukendrup** [comment]
Does this maybe also resolve #12821? It looks like that could be the same issue as #13159.

**Comment by MarcSabatella** [comment]
These are almost certainly all the same issue, except the text portion of #12835 which isn't reproducible anymore anyhow.

**Comment by MatheusWittmann** [comment]
I've just downloaded the last version available, and I can reproduce exactly this error :(

**Comment by MarcSabatella** [comment]
Right, this PR isn’t merged yet.

**Review by RomanPudashkin — APPROVED** [review]

**Pull request merged**